options/timeout.go

```diff
@@ -1,12 +1,37 @@
 package options
 
+import "reflect"
+
 type TimeoutOptions struct {
 	// RequestTimeout is total request timeout
 	RequestTimeout uint32 `yaml:"request_timeout,omitempty" json:"request_timeout,omitempty"`
 	// IdleTimeout is timeout for idle connection
 	IdleTimeout uint32 `yaml:"idle_timeout,omitempty" json:"idle_timeout,omitempty"`
 }
 
+func (o *Options) GetTimeoutOptions(path, method string) TimeoutOptions {
```

Comment:
I'd use the same naming there as others, `GetTimeoutOpts`

Reply:
Done